org/org.yaml

```diff
@@ -0,0 +1,62 @@
+admins:
+- abayer
+- bobcatfish
+- caniszczyk
+- dlorenc
+- googlebot
+- mchmarny
+- tekton-robot
+- thelinuxfoundation
+- vdemeester
+billing_email: info@cd.foundation
+company: ""
+default_repository_permission: read
+description: ""
+email: ""
+has_organization_projects: true
+has_repository_projects: true
+location: ""
+members:
+- a-roberts
+- afrittoli
+- akihikokuroda
+- AlanGreene
+- carlos-logro
+- CarolynMabbott
+- chmouel
+- danielhelfand
+- dibbles
+- dibyom
+- dwnusbaum
+- ekcasey
+- EliZucker
+- garethr
+- houshengbo
+- hrishin
+- iancoffey
+- jessm12
+- jkutner
+- joseblas
+- jromero
+- khrm
+- kimsterv
+- knative-prow-robot
```

Comment:
Yeah, I think `knative-prow-robot` is not needed anymore :stuck_out_tongue_closed_eyes:

Reply:
Makes sense. I'll send a PR to drop that robot after we get this setup. It can be our first test!